sdk/OoyalaSkinSDK/OoyalaSkinSDK/OOSkinViewController.m

```diff
@@ -55,12 +55,12 @@ - (instancetype)initWithPlayer:(OOOoyalaPlayer *)player
     LOG(@"Ooyala SKin Version: %@", OO_SKIN_VERSION);
     [self setPlayer:player];
     _skinOptions = skinOptions;
+    _skinConfig = [self getReactViewInitialProperties];
+    kSkinCofig = _skinConfig;
     _reactView = [[RCTRootView alloc] initWithBundleURL:skinOptions.jsCodeLocation
                                              moduleName:@"OoyalaSkin"
+                                      initialProperties:_skinConfig
```

Comment:
i know you did this, and i know this is in newer React Native libraries. are you sure that the React Native in package.json gets the correct version of R-N?